src/styles/components/_profile-upload.scss

```
@@ -0,0 +1,108 @@
+@import '../abstracts/_abstracts-dir';
+
+.profile-upload {
+  flex-direction: column;
+  box-sizing: border-box;
+
+  // TODO: Remove this when merging to the form
+  background-color: $light;
+
+  @include flex-center;
+
+  &__label {
+    margin-bottom: rem(16px);
+
+    cursor: pointer;
+
+    @include respond-above('lg') {
+      margin-bottom: rem(24px);
+    }
+
+    &--selected {
+      position: relative;
+
+      padding: rem(8px 12px);
+      border: solid 1px $mercury;
+      border-radius: rem(10px);
+
+      @include respond-above('lg') {
+        padding: rem(12px 16px);
+      }
+    }
+  }
+
+  &__input {
+    display: none;
+    visibility: hidden;
+  }
+
+  &__image {
+    width: rem(120px);
+    height: rem(120px);
+
+    @include respond-above('lg') {
+      width: rem(160px);
+      height: rem(160px);
+    }
+
+    @include respond-above('xl') {
+      width: rem(200px);
+      height: rem(200px);
+    }
+
+    &--selected {
+      box-shadow: 0 rem(3px 6px) 0 box-shadow-opacity(0.16);
+      border-radius: 50%;
+      object-fit: cover;
+    }
+  }
+
+  &__icon {
+    position: absolute;
+    bottom: rem(8px);
+    right: rem(8px);
+
+    width: rem(16px);
+    height: rem(16px);
+
+    @include respond-above('lg') {
+      bottom: rem(12px);
+      right: rem(12px);
+
+      width: rem(22px);
+      height: rem(22px);
+    }
+
+    @include respond-above('xl') {
+      bottom: rem(14px);
+      right: rem(14px);
+
+      width: rem(24px);
+      height: rem(24px);
+    }
+  }
+
+  &__caption {
+    color: #333333;
+
+    font-size: rem(8px);
+    font-style: italic;
+    line-height: 1.14;
+
+    @include respond-above('lg') {
+      font-size: rem(12px);
+    }
+
+    @include respond-above('xl') {
+      font-size: rem(14px);
+    }
+
+    & > span:first-child {
+      margin-right: rem(24px);
```

I think this one no need to specify as ```first-child``` since you are use only one ``span``
```suggestion
    & > span {
      margin-right: rem(24px);
```